src/pytest_inmanta_lsm/plugin.py

```diff
@@ -96,6 +114,9 @@ def remote_orchestrator(project: Project, request, remote_orchestrator_settings)
     user = get_opt_or_env_or(request.config, "inm_lsm_remote_user", "centos")
     port = get_opt_or_env_or(request.config, "inm_lsm_remote_port", "22")
     noclean = get_opt_or_env_or(request.config, "inm_lsm_noclean", "false").lower() == "true"
+    ssl = get_opt_or_env_or(request.config, "inm_lsm_ssl", "false").lower() == "true"
+    token = get_opt_or_env_or(request.config, "inm_lsm_token", None)
+    ca_cert = get_opt_or_env_or(request.config, "inm_lsm_ca_cert", None)
```

Comment:
* We should also raise an exception when `ca_cert` references a non-existing file. 
* It would also be good to translate `ca_cert` to an absolute path. That way changing the `cwd` somewhere in the code will not result in undesired behavior.